pkg/rc/farm.go

```diff
@@ -200,6 +202,28 @@ START_LOOP:
 	}
 }
 
+func (rcf *Farm) failsafe(rcFields []fields.RC) {
+	// FAILSAFES. If no RCs are scheduled, or there are zero replicas of anything scheduled, panic
+	if len(rcFields) == 0 {
+		rcf.alerter.Alert(alerting.AlertInfo{
+			Description: "No RCs have been scheduled",
+			IncidentKey: "no_rcs_found",
+		})
+		panic("No RCs are scheduled at all. Create one RC to enable the farm. Panicking to escape a potentially bad situation.")
```

Comment:
When we get around to documenting it we can say to use `p2-rctl` to schedule an RC before starting the farms.